docs/faq.md

```diff
@@ -197,6 +216,6 @@ A policy is a simple collection of rules and a definition of what to do with rul
 
 ### Do you have a visualization UI?
 
-Semgrep Team users can create custom dashboards and visualizations. Semgrep also supports posting results through [webhooks](https://semgrep.dev/docs/semgrep-cloud-platform/webhooks) to any JSON endpoint, so you can easily integrate it with your favorite visualization tool.
+Semgrep Cloud Platform users can create custom dashboards and visualizations. Semgrep also supports posting results through [webhooks](https://semgrep.dev/docs/semgrep-cloud-platform/webhooks) to any JSON endpoint, so you can easily integrate it with your favorite visualization tool.
```

Comment:
Not Semgrep AppSec Platform as you changed above?

Reply:
Hi, I realized that we haven't updated any of our collateral yet around calling it an Appsec platform. Maybe I should stick to SCP first?